lib: at_cmd_parser: support negative numbers (#3726)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-latest.rst

    * Updated to use the new FOTA (fota_v2) based on nRF Connect for Cloud.

>   * :ref:'at_cmd_parser' library:

Reviewer: * :ref:`at_cmd_parser_readme` library:

>     * The library can now parse AT command strings with negative numbers with the range of a int32_t.

Reviewer: ```suggestion
    * The library can now parse AT command strings with negative numbers in the range supported by the int32_t type.
```
